### PR TITLE
Polish translation of Tour of Scala: classes

### PR DESCRIPTION
**Pull request by kamilduda**

### CODE REVIEW

**Comment by SethTisue** [comment]
can you rebase to see if it fixes the CI failure...?

**Comment by kamilduda** [comment]
@jstefanska Could you review this? 😃

**Comment by SethTisue** [comment]
ping @jstefanska, if you're around/available

File: _pl/tour/classes.md [hunk 1]
```diff
@@ -6,49 +6,127 @@ discourse: false
...
 }
+
+val point1 = new Point(2, 3)
+point1.x  // 2
+println(point1)  // wyświetla (2, 3)
+```
+
+Klasa `Point` ma cztery składniki: zmienne `x` i `y` oraz metody `move` i `toString`.
+W przeciwieństwie do innych języków programowania, główny konstruktor zawiera się w sygnaturze klasy: `(var x: Int, var y: Int)`.
+Metoda `move` przyjmuje jako parametry dwie liczby całkowite i zwraca wartość `()` typu `Unit`, która nie niesie żadnych informacji.
```
Comment (jstefanska) [inline _pl/tour/classes.md]:
```suggestion
Metoda `move` przyjmuje jako parametry dwie liczby całkowite i zwraca wartość `()` typu `Unit`, która nie niesie ze sobą żadnych informacji.
```

File: _pl/tour/classes.md [hunk 2]
```diff
@@ -6,49 +6,127 @@ discourse: false
...
+
+val origin = new Point  // x i y są mają wartość 0
+val point1 = new Point(1)
+println(point1.x)  // wyświetla 1
+
 ```
 
-Klasy w Scali są sparametryzowane poprzez argumenty konstruktora. Powyższy kod wymaga podania dwóch argumentów konstruktora: `x` i `y`. Oba parametry są widoczne w zasięgu ciała klasy.
+W tej wersji klasy `Point`, `x` oraz `y` mają domyślną wartość `0` - dlatego nie jest wymagane przekazanie żadnych parametrów.
+Ponieważ jednak, konstruktor jest ewaluowany od lewej do prawej strony, jeżeli chcesz przekazać parametr tylko do argumentu `y` musisz określić nazwę tego parametru.
```
Comment (jstefanska) [inline _pl/tour/classes.md]:
```suggestion
Jednak z powodu tego, że konstruktor jest ewaluowany od lewej do prawej strony, jeżeli chcesz przekazać parametr tylko do argumentu `y`, musisz określić nazwę tego parametru.
```

**Pull request merged**